frontend/src/libs/components/profile-settings-sidebar/profile-settings-sidebar.tsx

```diff
@@ -0,0 +1,83 @@
+import { Card, Icon } from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+import { useAppSelector, useCallback, useState } from '#libs/hooks/hooks.js';
+import { type IconName } from '#libs/types/types.js';
+import { type UserAuthResponseDto } from '#packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+const settingsOptions: {
```

Comment:
It is a constant, follow acceptance criteria for it. Also create a separate type for item

Reply:
I did add a type for for profile settings option, but not sure what exactly you meant by the acceptance criteria for it, I went over the code quality list and the closes I found was B22 - Words are separated by underscores (UPPER_SNAKE_CASE)

Reply:
Yep, sure. But not resolved. And move it to separate file